### PR TITLE
Inference pipeline

### PR DESCRIPTION
**Pull request by rishabhshah13**
- Added inference pipeline
- Added different platform installation instructions
- 3 different modes of inference viz.  Single model, Multi model, Multi model state maintain

### CODE REVIEW

File: scripts/Model.py
Comment (leslie-dees) [inline scripts/Model.py:1]: Good reorganization

**Review by leslie-dees — APPROVED** [review]
Overall looks good. Can combine with existing frontend

**Review by zihanxing — APPROVED** [review]
Overall looks good after reorganization. Some conflicts need to be resolved before merging though.

**Review by h0ngxuanli — APPROVED** [review]
Well-organized code

**Pull request merged**